sql/mob_pets.sql

```diff
@@ -37,1185 +39,1185 @@ CREATE TABLE `mob_pets` (
+INSERT INTO `mob_pets` VALUES (16973829,1,9,'Mamool_Ja_Handler','Mamool_Jas_Lizard');
+INSERT INTO `mob_pets` VALUES (16973838,1,9,'Mamool_Ja_Handler','Mamool_Jas_Lizard');
+INSERT INTO `mob_pets` VALUES (16973847,1,9,'Mamool_Ja_Handler','Mamool_Jas_Lizard');
+INSERT INTO `mob_pets` VALUES (16973903,1,9,'Thunderclap_Sareel_Ja','Thunderbolt_Piraal_Ja');
+INSERT INTO `mob_pets` VALUES (16974027,1,15,'Lamia_Commandress','Lamias_Elemental');
+INSERT INTO `mob_pets` VALUES (16974036,1,15,'Lamia_Commandress','Lamias_Elemental');
+INSERT INTO `mob_pets` VALUES (16974045,1,15,'Lamia_Commandress','Lamias_Elemental');
+INSERT INTO `mob_pets` VALUES (16974108,1,15,'Lamia_No9','Lamias_Elemental');
+INSERT INTO `mob_pets` VALUES (16986133,1,9,'Fomor_Beastmaster','Fomors_Bats');
+INSERT INTO `mob_pets` VALUES (16986164,1,9,'Fomor_Beastmaster','Fomors_Bats');
+INSERT INTO `mob_pets` VALUES (16986197,1,9,'Zoraal_Jas_Pkuucha','Percipient_Zoraal_Ja');
+INSERT INTO `mob_pets` VALUES (16990223,1,9,'Fomor_Beastmaster','Fomors_Bats');
+INSERT INTO `mob_pets` VALUES (16990232,1,9,'Fomor_Beastmaster','Fomors_Bats');
+INSERT INTO `mob_pets` VALUES (16990247,1,9,'Fomor_Beastmaster','Fomors_Bats');
+INSERT INTO `mob_pets` VALUES (16990319,1,18,'Troll_Surveillant','Trolls_Automaton');
+INSERT INTO `mob_pets` VALUES (16990323,1,18,'Troll_Surveillant','Trolls_Automaton');
+INSERT INTO `mob_pets` VALUES (16990386,1,9,'Mamool_Ja_Stabler','Mamool_Jas_Raptor');
+INSERT INTO `mob_pets` VALUES (16990396,1,14,'Mamool_Ja_Pikeman','Mamool_Jas_Wyvern');
+INSERT INTO `mob_pets` VALUES (16990405,1,9,'Mamool_Ja_Stabler','Mamool_Jas_Raptor');
+INSERT INTO `mob_pets` VALUES (16990410,1,14,'Mamool_Ja_Pikeman','Mamool_Jas_Wyvern');
+INSERT INTO `mob_pets` VALUES (16990435,1,9,'Mamool_Ja_Stabler','Mamool_Jas_Raptor');
+INSERT INTO `mob_pets` VALUES (16990437,1,9,'Mamool_Ja_Stabler','Mamool_Jas_Raptor');
+INSERT INTO `mob_pets` VALUES (16990440,1,14,'Mamool_Ja_Pikeman','Mamool_Jas_Wyvern');
+INSERT INTO `mob_pets` VALUES (16990442,1,14,'Mamool_Ja_Pikeman','Mamool_Jas_Wyvern');
+INSERT INTO `mob_pets` VALUES (16998427,1,15,'Lamia_Dancer','Lamias_Elemental');
+INSERT INTO `mob_pets` VALUES (16998440,1,14,'Reserve_Draugar','Draugars_Wyvern');
+INSERT INTO `mob_pets` VALUES (16998444,1,14,'Reserve_Draugar','Draugars_Wyvern');
+INSERT INTO `mob_pets` VALUES (16998456,1,14,'Reserve_Draugar','Draugars_Wyvern');
+INSERT INTO `mob_pets` VALUES (16998518,1,15,'Lamia_Dancer','Lamias_Elemental');
+INSERT INTO `mob_pets` VALUES (16998534,1,14,'Reserve_Draugar','Draugars_Wyvern');
+INSERT INTO `mob_pets` VALUES (16998540,1,14,'Reserve_Draugar','Draugars_Wyvern');
+INSERT INTO `mob_pets` VALUES (16998636,1,15,'Lamia_Bellydancer','Lamias_Elemental');
+INSERT INTO `mob_pets` VALUES (16998648,1,14,'Draugar_Servant','Draugars_Wyvern');
+INSERT INTO `mob_pets` VALUES (16998659,1,14,'Draugar_Servant','Draugars_Wyvern');
+INSERT INTO `mob_pets` VALUES (16998742,1,15,'Lamie_No9','Lamias_Avatar');
```

Comment:
Should this be Lamia and not Lamie?  (Also two other instances below)  Other than that, I really like this 👍